doc/architecture/computing-services.md

```diff
@@ -0,0 +1,105 @@
+
+Table of Contents
+=================
+
+   * [Table of Contents](#table-of-contents)
+   * [Motivation](#motivation)
+   * [Architecture](#architecture)
+      * [Enabling Publisher Services (Brizo)](#enabling-publisher-services-brizo)
+      * [Responsibilities](#responsibilities)
+      * [Initial solution](#initial-solution)
+      * [Next steps](#next-steps)
+
+
+---
+
+This page describes the highlights of the integration of additional computing services.
+
+# Motivation
+
+The most basic scenario of a Publisher, is to provide access to the Assets this Publisher owns or manage. In addition to this, having this data ecosystem in place,
+Publishers could offer other related services in top of this. Some possibles scenarios are:
+
+* The Publishers offer as a service the possibility of execute some computation on top of their data. This has some benefits:
+  - The data never leaves the Publisher enclave
+  - It's not necessary to move the data, the algorithm moves to the data
+  - Regulation
+
+* Storage services for new derived assets
```

Comment:
I don't understand why Ocean needs to provide special services to store _derived_ assets. I thought the general philosophy was to let people store their assets/data (or derived data) wherever they like, however they like. I thought Ocean only provided services for storing metadata.

Maybe this is necessary because the Publisher wants to control where the derived assets get stored? If so, that should be explained.